app/src/pages/Lobby/Inputs/index.tsx

```diff
@@ -39,16 +41,19 @@ export function LetterInput(props: { value: string }) {
 
   return (
     <TextField
-      label="Letter"
+      label={t("settings.cards.letter.label", "Letter")}
```

Comment:
im impressed by the namespacing haha, will these just remain one key or get nested in json outputs for other languages?

Reply:
Yeah, it's tedious but mostly necessary – otherwise your keys are going to conflict, even in a small app like this. It also gives translators context (i.e. groups related strings).

> will these just remain one key or get nested in json outputs for other languages?

It depends on the resource file format, but yeah for JSON (which is served over the CDN), the dots will result in object nesting.